File: PRAKTIKUM 3/Bahasa C/PRAK305-2510817120014-NailaAtresiaRahma.c
#include <stdio.h>

int main() { 

    int hari, jam, menit, detik, x;
    scanf("%d", &x);

    hari= x / (24*3600);
    jam= x % (24*3600)/3600;
    menit= x % 3600/60;
    detik= x %60;

    if(x<=86400) {
    printf("%02d:%02d:%02d", jam, menit, detik);
    } else if(x>86400) {
        printf("%d hari %02d:%02d:%02d", hari, jam, menit, detik);
    }

    return 0;
}

File: PRAKTIKUM 3/Bahasa Python/PRAK305-2510817120014-NailaAtresiaRahma.py
x = int(input())

hari= x/(24*3600)
jam= x % (24*3600)/3600
menit= x % 3600/60
detik= x % 60

if x<=86400:
    print("%02d:%02d:%02d" % (jam, menit, detik))
elif x>86400:
    print("%d hari %02d:%02d:%02d" % (hari, jam, menit, detik))